mautic/mautic
# Do Not Contact Plugin

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | **Y**
| Automated tests included? | N
| Related user documentation PR URL | N
| Related developer documentation PR URL | N
| Issues addressed (#s or URLs) | N
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
With this plugin, you can put the action of adding or removing a contact from the "Do Not Contact" status of a chosen channel.

#### Steps to test this PR:
1. Create new campaign.
2. Add new action, in the action list, the two new options appear.
